spec/retriable_spec.rb

```diff
@@ -313,6 +313,34 @@ class SecondTestError < TestError; end
       expect(exceptions[3].class).must_equal StandardError
     end
 
+    it '#retriable retries with a hash exception where the value is a proc ' \
+       'that returns true' do
```

Comment:
not sure why this part isn't just on the previous line? there's plenty of room even given the style guide limitations